src/effects/VST3/VST3Effect.cpp

```diff
@@ -361,10 +361,13 @@ bool VST3Effect::SaveSettings(
       {
          if (parameterInfo.flags & Vst::ParameterInfo::kCanAutomate)
          {
-            parms.Write(
-               VST3Utils::MakeAutomationParameterKey(parameterInfo),
-               ownSettings.mValues.at(parameterInfo.id)
-            );
+            const auto itr = ownSettings.mValues.find(parameterInfo.id);
+
+            if (itr != ownSettings.mValues.end())
+            {
+               parms.Write(VST3Utils::MakeAutomationParameterKey(parameterInfo),
+                           ownSettings.mValues.at(parameterInfo.id));
+            }
          }
```

Comment:
woops I just realized that I am searching the map two times - instead of .at() I should have used the iterator directly.